src/components/todo/TodoForm.tsx

```diff
@@ -13,7 +13,12 @@ const TodoForm = ({ todos, setTodos }: Props) => {
     e.preventDefault();
     if (todotext) {
       const updatedTodos = [...todos];
-      const todo = { id: Date.now(), todo: todotext, isDone: false };
+      const todo = {
+        id: Date.now(),
```

Comment:
We have ``moment``, we can use that here.

Reply:
updated code and used moment

Reply:
updated